test-cache/models/Cache-Address.spec.ts

```diff
@@ -0,0 +1,138 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017 NEM
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+import {expect} from "chai";
+import { CacheAccount } from '../../src/cache-model/cache-account/Cache-Account';
+import { CacheAddress } from '../../src/cache-model/cache-account/Cache-Address';
+import {XEM} from "../../src/models/mosaic/XEM";
+import {NetworkTypes} from "../../src/models/node/NetworkTypes";
+import {MultisigSignatureTransaction} from "../../src/models/transaction/MultisigSignatureTransaction";
+import {EmptyMessage, PlainMessage} from "../../src/models/transaction/PlainMessage";
+import {TimeWindow} from "../../src/models/transaction/TimeWindow";
+import {TransferTransaction} from "../../src/models/transaction/TransferTransaction";
+import {NEMLibrary} from "../../src/NEMLibrary";
+import { TestVariables } from '../../test/config/TestVariables.spec';
+
+declare let process: any;
+
+describe("CacheAccount", () => {
+  const recipientAccount = new CacheAddress(TestVariables.TEST_ADDRESS);
+  const publicKey: string = TestVariables.TEST_PUBLIC_KEY;
+  const privateKey: string = TestVariables.TEST_PRIVATE_KEY;
+  const undefinedPrivateKey: string = process.env.NOT_DEFINED_PRIVATE_KEY;
+
+  before(() => {
+    NEMLibrary.bootstrap(NetworkTypes.TEST_NET);
+  });
+
+  after(() => {
+    NEMLibrary.reset();
+  });
+
+  it("should sign a multisig transaction", () => {
+    const account = new CacheAccount(recipientAccount, publicKey, privateKey);
+    const transaction = new MultisigSignatureTransaction(
```

Comment:
Is this test really needed? I think this code isn't testing CacheAccount/CacheAddress as much as it is testing multisig which was already tested in other places by Aleix. What do you think?